### PR TITLE
UT fixes

### PR DESCRIPTION
**Pull request by rohitcbr**
### - Link added should be a valid URL and accessible throughout the org.
   ### - If the branch name contains hotfix / revert by default the BVT workflow check will pass.

| Test Case Document URL                        |
|-----------------------------------------------|
| Please paste test case document link here.... |

### CODE REVIEW

File: includes/api/api.php [hunk 1]
```diff
@@ -179,6 +179,13 @@ function initCartCommon()
 
 }
 
+function initAdminFunctions()
```
Comment (yashgit891) [inline includes/api/api.php:182]: @rohitcbr this is file related to 1cc is the change related to a 1cc test?

File: includes/razorpay-affordability-widget.php [hunk 2]
```diff
@@ -505,17 +505,18 @@ function getAffordabilityWidgetSettings()
...
+    }
 
     return apply_filters('wc_affordability_widget_settings', $settings);
 }
 
-function displayAffordabilityWidgetSettings() 
+function displayAffordabilityWidgetSettings()
 {
-    woocommerce_admin_fields(getAffordabilityWidgetSettings()); 
+    initAdminFunctions();
```
Comment (yashgit891) [inline includes/razorpay-affordability-widget.php:515]: @rohitcbr is this change tested?

File: woo-razorpay.php [hunk 3]
```diff
@@ -82,9 +82,7 @@ function() {
 
 function woocommerce_razorpay_init()
 {
-    add_action("woocommerce_update_options_advanced", 'hposInstrumentation');
-
-    function hposInstrumentation()
+    add_action("woocommerce_update_options_advanced", function()
```
Comment (yashgit891) [inline woo-razorpay.php:85]: @rohitcbr has this been tested for any event?

**Review by yashgit891 — APPROVED** [review]

File: woo-razorpay.php (hunk 3 above)
Comment (ramth05) [reply woo-razorpay.php:85]: @rohitcbr , why this change required?

File: includes/razorpay-affordability-widget.php [hunk 4]
```diff
@@ -505,17 +505,28 @@ function getAffordabilityWidgetSettings()
...
 {
-    woocommerce_admin_fields(getAffordabilityWidgetSettings()); 
+    if (defined('WC_ABSPATH')) {
+        include_once WC_ABSPATH . 'includes/admin/wc-admin-functions.php';
+    }
+}
+
+function displayAffordabilityWidgetSettings()
+{
+    if(function_exists('woocommerce_admin_fields') === false)
```
Comment (abdulwahidsharief) [inline includes/razorpay-affordability-widget.php:522]: @rohitcbr leave space after if

File: includes/razorpay-affordability-widget.php [hunk 5]
```diff
@@ -505,17 +505,28 @@ function getAffordabilityWidgetSettings()
...
+    {
+        initAdminFunctions();
```
Comment (abdulwahidsharief) [inline includes/razorpay-affordability-widget.php:524]: @rohitcbr as suggested by ram can we include logic here instead of creating initAdminFunctions()

**Review by abdulwahidsharief — APPROVED** [review]
LGTM

**Pull request merged**